.github/workflows/main.yml

```diff
@@ -36,6 +36,43 @@ jobs:
       uses: coverallsapp/github-action@master
       with:
         github-token: ${{ secrets.GITHUB_TOKEN }}
+  PublishGithub:
+    if: github.ref == 'refs/heads/master' && startsWith(github.repository, 'neo-project/')
+    needs: Test
+    runs-on: ubuntu-latest
+    steps:
+    - name: Chectout
+      uses: actions/checkout@v4
+
+    - name: Setup .NET Core
+      uses: actions/setup-dotnet@v4
+      with:
+        dotnet-version: ${{ env.DOTNET_VERSION }}
+
+    - name: Set Version
+      run: git rev-list --count HEAD | xargs printf 'CI%05d' | xargs -I{} echo 'VERSION_SUFFIX={}' >> $GITHUB_ENV
+
+    - name : Pack (Neo)
+      run: |
+        dotnet pack \
+        --configuration Debug \
```

Comment:
Is it OK to have Debug configuration in the resulting package?

Reply:
Yes. This would for the our testing. We can't use `MyGet` anymore for some reason.

Reply:
I think that it should be Release, we don't need packages for our tests

Reply:
You got it